PWGDQ/Tasks/dileptonMuMu.cxx

```diff
@@ -286,13 +304,15 @@ struct DQEventMixing {
             continue;
           }
           VarManager::FillPair(muon1, muon2, fValues, VarManager::kJpsiToMuMu);
+          int centBin = fMixHandler->GetBinFromCategory(fMixHandler->GetMixingVariable("Cent"), event1.mixingHash());
+          //            std::cout<<"  CentralityBin "<< centBin<<std::endl;
           if (muon1.sign() * muon2.sign() < 0) {
-            fHistMan->FillHistClass("PairsMuonMEPM", fValues);
+            fHistMan->FillHistClass(Form("PairsMuonMEPM_%s", fCentBinNames[centBin].Data()), fValues);
```

Comment:
I am not sure i understand the logic here. Why do you want to fill histogram directories for separate centrality ranges? You can have one single histogram directory with histograms which have centrality as one of their dimensions.

Reply:
I simply didn't think of using a 2D histogram. In Run2, I used to have a lot of 1D histograms instead of 2D or 3D histograms, that were more memory consuming, and I tend to still think like that. A histogram with centrality as a dimension is a more efficient way though, I agree.